docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -1,7 +1,7 @@
 ---
 productId: toolpad-core
 title: Dashboard Layout
-components: AppProvider, DashboardLayout
+components: AppProvider, DashboardLayout, Account
```

Comment:
Needs a section and demo on the user button. And an explanation on how to customize it (slots?).